lang/cs/Org.Apache.REEF.Wake/Org.Apache.REEF.Wake.csproj

```diff
@@ -27,9 +27,17 @@ under the License.
     <FileAlignment>512</FileAlignment>
     <RestorePackages>true</RestorePackages>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..</SolutionDir>
+    <AvroBinaryDirectory>..\packages\AvroBin</AvroBinaryDirectory>
```

Comment:
I think it is because build.prop is imported after this session. 

For the tool versions, you can defined them in the build.prop and use them in the csproj so that we can maintain the version at one place.